backend/src/main/java/se/chalmers/dat265/group1/model/dto/DigitalPrintMagics.java

```diff
@@ -0,0 +1,14 @@
+package se.chalmers.dat265.group1.model.dto;
+
+import se.chalmers.dat265.group1.model.DigitalPrint;
+
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)